source/slang/slang-check-expr.cpp

```diff
@@ -1509,18 +1509,43 @@ namespace Slang
         return expr;
     }
 
+    // This function proceses primal arguments that need to be carried over to the function
```

Comment:
What is the definition for "primal" arguments? Would be good to define it more clearly in the comment.

Reply:
Adjusted the comment